packages/sources/cfbenchmarks-test/src/endpoint/websocket/crypto.ts

```diff
@@ -0,0 +1,150 @@
+import { makeLogger } from '@chainlink/external-adapter-framework/util'
+import {
+  WebSocketTransport,
+  WebSocketRawData,
+} from '@chainlink/external-adapter-framework/transports/websocket'
+import { WebSocket } from '@chainlink/external-adapter-framework/transports/websocket'
+import { EndpointTypes, getIdFromBaseQuote, getBaseQuoteFromId } from '../common/crypto'
+
+interface Message {
+  type: 'subscribe' | 'unsubscribe' | 'value'
+  id: string
+  value: string
+  time: number
+}
+export interface WsErrorType {
+  TYPE: string
+  MESSAGE: string
+  PARAMETER: string
+  INFO: string
+}
+
+export type WsEndpointTypes = EndpointTypes & {
+  Provider: {
+    WsMessage: Message
+  }
+}
+
+const logger = makeLogger('CfbenchmarksCryptoWebsocketEndpoint')
+
+export const transport = new WebSocketTransport<WsEndpointTypes>({
```

Comment:
Same comment as in the batch ones, these two ws transports are identical save for 1 string

Reply:
3 lines achstually :nerd_face: 

Agreed. Done.